src/pages/Community/Community.jsx

```diff
@@ -1,7 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Category } from './Component/Category.jsx';
+import Article from './Component/Article.jsx';
+import Detail from './Component/Detail.jsx';
+import Writing from './Component/Writing.jsx';
+import Debate from './Component/Debate.jsx';
+import DebateDetail from './Component/DebateDetail.jsx';
+import Button from '../../components/Button/Button.jsx';
+import Dummy from './Component/Dummy.jsx';
+import * as Com from './Community.js';
 
 const Community = () => {
-  return <div>Community</div>;
+  const { id } = useParams();
+  const [reviews, setReviews] = useState([]);
+  const [value, setValue] = useState('');
+  const articleIds = ['1', '2', '3', '4'];
+
+  // const saveValue = e => {
+  //   console.log(e.target.value);
+  //   return setValue(e.target.value);
+  // };
+
+  useEffect(() => {
+    if (id === '1') {
+      setReviews(Dummy);
+    } else if (id === '2') {
+      setReviews(() => {
+        return Dummy.filter(dummy => dummy.category === '공연리뷰');
+      });
+    } else if (id === '3') {
+      setReviews(() => {
+        return Dummy.filter(dummy => dummy.category === '공연장리뷰');
+      });
+    } else if (id === '4') {
+      setReviews(() => {
+        return Dummy.filter(dummy => dummy.category === '공연홍보');
+      });
+    }
+  }, [id]);
+
```

Comment:
```suggestion
const object = {
    5 : <Debate />
    6 : <Writing />
}
```
```
{ object[id] }
```
이렇게 계산된 속성명으로 조건부 랜더링을 할 수 있습니다